tests/karma/unit/enketo/phone-widget.js

```diff
@@ -0,0 +1,116 @@
+describe('phone-widget', function() {
```

Comment:
I would strongly recommend following the enketo code formatting in enketo widgets - most of the code tends to be boilerplate, and comparing the boilerplate with upstream won't be easy if the formatting's different.